.github/tests.yaml

```diff
@@ -0,0 +1,31 @@
+betelgeuse: "tests/foreman/ -v -m 'build_sanity'"
+broker[docker]: "tests/foreman/ -v -m 'build_sanity'"
+cryptography: "tests/foreman/ -v -m 'build_sanity'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -v -k 'test_positive_get_links'"
```

Comment:
are we not going to rely on robottelo dependency tests for this? why to run build_sanity for all dependancies?

Reply:
Yes either run direct tests using this dependency from robottelo or run unint tests.

Reply:
I'd vote for unit tests as those were written specifically to test these requirement bumps, and could save time when we tend to run the entire tests for small version bumps in master and then again for cpick PRs :)

Reply:
as @jyejare mentioned we wanted to run the robottelo API/UI/CLI test if the package is used else run unit tests. As the unit tests are already run part of CI checks no need to pass them there.